scene/resources/navigation_polygon.cpp

```diff
@@ -347,6 +347,15 @@ real_t NavigationPolygon::get_cell_size() const {
 	return cell_size;
 }
 
+void NavigationPolygon::set_border_size(real_t p_value) {
+	ERR_FAIL_COND(p_value < 0.0);
+	border_size = p_value;
```

Comment:
shouldn't we `emit_changed();` here as well?

Reply:
I did not add a changed signal for this property because all the other properties that do not need to update the inspector or the internal navigation mesh or some debug have a changed signal.

Reply:
Sounds good then!